src/index.ts

```diff
@@ -1,12 +1,50 @@
-import express, { Application, Request, Response } from 'express';
+import express, { Application, Request, Response, NextFunction} from 'express';
 
-let indexRouter = require('./routes/home');
+const dotenv = require('dotenv');
+dotenv.config()
+
+const createError = require('http-errors');
+const path = require('path');
+
+const indexRouter = require('./routes/main');
+const badgeclassRouter = require('./routes/badgeclass')
+const assertionRouter = require('./routes/assertion')
 
 const PORT = 5000;
```

Comment:
Let's check if a `PORT` environment variable is set and default to 5000.

```suggestion
const PORT = process.env.PORT || 5000;
```